sycl/source/detail/event_impl.cpp

```diff
@@ -128,7 +129,8 @@ event_impl::get_profiling_info<info::event_profiling::command_submit>() const {
         this->getHandleRef(), this->getPlugin());
   }
   if (!MHostProfilingInfo)
-    throw invalid_object_error("Profiling info is not available.");
+    throw invalid_object_error("Profiling info is not available.",
+                               PI_INVALID_DEVICE);
```

Comment:
Something like CL_PROFILING_INFO_NOT_AVAILABLE ?

Reply:
Done. There is PI_PROFILING_INFO_NOT_AVAILABLE now.